EduNLP/Formula/Formula.py

```diff
@@ -135,6 +148,10 @@ class FormulaGroup(object):
     [{'id': 0, 'type': 'mathord', 'text': 'x', 'role': None, 'var': 0}, \
 {'id': 1, 'type': 'mathord', 'text': 'y', 'role': None, 'var': 1}, \
 {'id': 2, 'type': 'mathord', 'text': 'x', 'role': None, 'var': 0}]
+
+    Returns
+    --------
+    the parsed abstracted syntax forest
```

Comment:
For differentiating Formula and Formula Group, in my view, we can define the description in the head of the comments, instead of the return.